--- ./config/Config-devel.in	2015-07-09 19:49:34.000000000 +0800
+++ ./config/Config-devel.in	2015-07-09 20:45:16.324985647 +0800
@@ -95,6 +95,6 @@ menuconfig DEVEL
 
 	config EXTRA_OPTIMIZATION
 		string "Additional compiler options" if DEVEL
-		default "-fno-caller-saves"
+		default "-fno-caller-saves -fno-caller-saves -fomit-frame-pointer -freorder-blocks"
 		help
 		  Extra Target-independent optimizations to use when building for the target.
